src/ctap/storage.rs

```diff
@@ -231,22 +229,29 @@ impl PersistentStore {
                 })
                 .unwrap();
         }
-        // The following 3 entries are meant to be written by vendor-specific commands.
-        if USE_BATCH_ATTESTATION {
-            if self.store.find_one(&Key::AttestationPrivateKey).is_none() {
-                self.set_attestation_private_key(key_material::ATTESTATION_PRIVATE_KEY)
-                    .unwrap();
-            }
-            if self.store.find_one(&Key::AttestationCertificate).is_none() {
-                self.set_attestation_certificate(key_material::ATTESTATION_CERTIFICATE)
-                    .unwrap();
-            }
-        }
+        // TODO(jmichel): remove this when vendor command is in place
+        #[cfg(not(any(test, feature = "ram_storage")))]
+        self.load_attestation_from_firmware();
+
         if self.store.find_one(&Key::Aaguid).is_none() {
             self.set_aaguid(key_material::AAGUID).unwrap();
         }
     }
 
+    // TODO(jmichel): remove this function when vendor command is in place.
+    #[cfg(not(any(test, feature = "ram_storage")))]
```

Comment:
Just wondering, but do we still need this `ram_storage` feature? Asking because it uses "testing" code. So we could have a custom module (different from the one we use in testing) if we want to reduce memory footprint.

Reply:
I don't know. I was simply copying the conditions used above to ensure all the test were working as expected.

Reply:
Ok there's 2 questions then :-)

1. Do we still need the `ram_storage` feature in general? Unrelated to this PR.
2. When do we want to load the baked-in values? If it's in prod keys, then we should just do `not(test)`